tests/run-make/static-pie/rmake.rs

```diff
@@ -0,0 +1,48 @@
+// How to manually run this
+// $ ./x.py test --target x86_64-unknown-linux-[musl,gnu] tests/run-make/static-pie
+
+//@ only-x86_64
+//@ only-linux
+//@ ignore-32bit
+
+use std::process::Command;
+
+use run_make_support::llvm_readobj;
+use run_make_support::rustc;
+use run_make_support::{cmd, env_var};
+
+fn ok_compiler_version(compiler: &str) -> bool {
+    let check_file = format!("check_{compiler}_version.sh");
+
+    Command::new(check_file).status().is_ok_and(|status| status.success())
+}
+
+fn test(compiler: &str) {
+    if !ok_compiler_version(compiler) {
+        return;
+    }
+
+    let target = env_var("TARGET");
+
+    rustc()
+        .input("test-aslr.rs")
+        .target(&target)
+        .arg(format!("-Clinker={compiler}"))
+        .arg("-Clinker-flavor=gcc")
+        .arg("-Ctarget-feature=+crt-static")
```

Comment:
Suggestion: we should add helpers for these to the support library if they don't already exist.